code/modules/economy/price_list.dm

```diff
@@ -1031,15 +1031,6 @@
 /obj/item/organ
 	price_tag = 500
 
-/mob/living/carbon/superior_animal/roach
-	price_tag = 150
-
-/mob/living/carbon/superior_animal/roach/roachling
-	price_tag = 100
-
-/mob/living/carbon/superior_animal/roach/hunter
-	price_tag = 200
```

Comment:
Are unprocessed roaches going to be factored in now that you removed this part?

Reply:
What?